packages/i18n/src/locales/en.i18n.json

```
@@ -282,6 +282,8 @@
   "Accounts_TwoFactorAuthentication_By_Email_Code_Expiration": "Time to expire the code sent via email in seconds",
   "Accounts_TwoFactorAuthentication_By_Email_Enabled": "Enable Two Factor Authentication via Email",
   "Accounts_TwoFactorAuthentication_By_Email_Enabled_Description": "Users with email verified and the option enabled in their profile page will receive an email with a temporary code to authorize certain actions like login, save the profile, etc.",
+  "Accounts_twoFactorAuthentication_email_available_for_oAuth_users": "Make two factor via email available for oAuth users",
+  "Accounts_twoFactorAuthentication_email_available_for_oAuth_users_Description": "People that use oAuth will receive an email with a temporary code to authorize actions like login, save profile, etc.",
```

```suggestion
  "Accounts_twoFactorAuthentication_email_available_for_OAuth_users": "Make two factor via email available for OAuth users",
  "Accounts_twoFactorAuthentication_email_available_for_OAuth_users_Description": "People that use OAuth will receive an email with a temporary code to authorize actions like login, save profile, etc.",
```
Kinda picky, but can we please update this? :grimacing: That's how we "spell" it everywhere else

---

Done! 🙂